media-gfx/blender/blender-2.83.4.ebuild

```diff
@@ -147,34 +154,41 @@ src_configure() {
 	# shadows, see bug #276338 for reference
 	append-flags -funsigned-char
 	append-lfs-flags
-	# Blender is compatible ABI 4 or less, so use ABI 4.
-	append-cppflags -DOPENVDB_ABI_VERSION_NUMBER=4
+
+	local version
+	if use abi6-compat; then
+		version=6;
+	elif use abi7-compat; then
+		version=7;
+	else
+		die "Openvdb abi version not compatible"
+	fi
+	append-cppflags -DOPENVDB_ABI_VERSION_NUMBER=${version}
 
 	local mycmakeargs=(
 		-DPYTHON_VERSION="${EPYTHON/python/}"
 		-DPYTHON_LIBRARY="$(python_get_library_path)"
 		-DPYTHON_INCLUDE_DIR="$(python_get_includedir)"
 		-DWITH_INSTALL_PORTABLE=OFF
-		-DWITH_PYTHON_INSTALL=OFF
-		-DWITH_PYTHON_INSTALL_NUMPY=OFF
+		-DWITH_PYTHON_INSTALL=$(usex system-python OFF ON)
+		-DWITH_PYTHON_INSTALL_NUMPY=$(usex system-numpy OFF ON)
 		-DWITH_STATIC_LIBS=OFF
+		-DBUILD_SHARED_LIBS=OFF
```

Comment:
This looks odd. Does nothing use it?

Reply:
WITH_STATIC_LIBS tries to make a portable blender, but most people install it on their system so I have not created a USE flag for it. It will require static libraries for opencollada and boost among others.

BUILD_SHARED_LIBS was introduced to break cyclic dependencies that occurred when compiling the cycles render engine.